mailarchive/mailarchive-tool/tool/src/webapp/vm/mailarchive/chef_mailbox-view.vm

```diff
@@ -53,7 +53,7 @@
 		#end
 		<tr>
 			<th>$tlang.getString("gen.bod2")</th>
-			<td>$validator.escapeHtmlFormattedText($email.FormattedBody)</td>
```

Comment:
getHtmlBody only returns the content of the body. This method might still be needed in combination with that other one still? I would test to make sure this doesn't introduce XSS in a html body.

Reply:
It's probably fine, it looks like it cleans it when it inserts it via SAK-32322

Reply:
There are no commits on SAK-32322.

But yeah I agree with just modifying addToArchive

Reply:
Sorry, I meant SAK-30197

Reply:
I remember going back and forth a lot with @joserabal on that one because of some edge cases and having to put it in that method.